iroh-net/src/netcheck.rs

```diff
@@ -686,64 +682,6 @@ impl Actor {
 
         r
     }
-
-    fn log_concise_report(&self, r: &Report, dm: &DerpMap) {
```

Comment:
why no nice report log anymore?

Reply:
I think this was mostly a go thing, this was essentially a manual implementation of Rust's `Debug`.  So I switched it to `Debug`, we get the same information equally clearly.

Reply:
ok